ios/browser/api/certificate/utils/brave_certificate_utils.mm

```diff
@@ -0,0 +1,74 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#import "brave/ios/browser/api/certificate/utils/brave_certificate_utils.h"
+
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
+
+namespace brave {
+NSData* string_to_data(const std::string& str) {
```

Comment:
The Chromium style for utility conversion methods usually do upper camel case instead of snake case, such as:

- `base::SysUTF8ToNSString`
- `base::Time::FromNSDate`
- `net::GURLWithNSURL`